cypress/integration/page.spec.js

```diff
@@ -509,7 +538,7 @@
     describe('if the other user is selected', () => {
       beforeEach(() => {
         page.mockUsers();
-        page.mockUser1Posts()
+        page.mockUser1Posts();
```

Comment:
The `page.mockUser1Posts()` call should be followed by `.as('user1PostsRequest')`.